integration_testing/features/coach/coach-learners-report.feature

```diff
@@ -0,0 +1,29 @@
+Feature: Coach learners report
+    Coach should be able to access Kolibri Coach tab
+    facility admin should be assign classes to coach facility to access the coach tab
+    Coach can look at each of the report types for the progress you have just logged
+
+  Background:
+    Given there is a learner account with name "Hansen" in class "Buffoons"
+    And there is a channel with name "Pratham Books' StoryWeaver" with topic "Night Trouble" that  contains exercises
+    And learner account named "Hansen" has completed all the exercises under topic "Night Trouble" containing topic "Pratham Books' StoryWeaver"
+    Given I am logged in as a coach for the facility
+
+  Scenario: Reviewing exercise completion of a particular student in a class for a topic
+    When I click on the Coach tab in the sidebar
+    Then I see a list of classes (as well as an "All Learners" link)
```

Comment:
Could you review this step? There is no `"All Learners" link` in release 0.10, or at least I can't see one on the public demo server https://kolibridemo.learningequality.org/coach/#/